firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionViewController.swift

```diff
@@ -150,8 +150,10 @@ class TrackingProtectionViewController: UIViewController, Themeable, Notifiable,
         updateProtectionViewStatus()
         applyTheme()
         getCertificates(for: viewModel.url) { [weak self] certificates in
-            if let certs = certificates {
-                self?.viewModel.certificates = certs
+            Task { [weak self] in
+                if let certificates {
+                    await self?.viewModel.certificates = certificates
+                }
```

Comment:
This seems fairly significant, am I misreading or is this now potentially updating certificates off the MT? What was the behavior previously?

Reply:
Yes, you're right the UI is updated potentially off the MT as was already implemented. I'll update it.